### PR TITLE
Filter services and users by province

### PR DESCRIPTION
**Pull request by olya-morozova**
## Description

Allows filtering services and users by location assignments in the specified province.

## How has this been tested?

Expanded on existing tests.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [x] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -756,6 +761,10 @@ Set a filter which will tell the API to return a user matching the provided iden
 
 Send the API request using the pre-set filters.
 
+- `located(details: LocatableServiceParameters)`
```
Comment (craigpaul) [inline README.md:764]: This doesn't seem like the correct type. I believe this should be `LocatableUserParameters` correct?

File: src/resources/service.test.ts [hunk 2]
```diff
@@ -223,3 +233,46 @@ it('can conditionally not set a filter', async () => {
...
+  expect(mockAxios.get).toHaveBeenCalledTimes(1);
+  expect(mockAxios.get).toHaveBeenCalledWith('services', {
+    params: {
+      'filter[location]': 1,
+      limit: 5,
+      page: 1,
+      sort: 'created',
+    },
+  });
+});
```
Comment (craigpaul) [inline src/resources/service.test.ts:257]: What is the reasoning behind not supplying both parameters if they are both supplied? I don't think this package should be laying down rules like this.

File: src/resources/user.test.ts [hunk 3]
```diff
@@ -196,3 +206,46 @@ it('can conditionally not set a filter', async () => {
...
+  expect(mockAxios.get).toHaveBeenCalledTimes(1);
+  expect(mockAxios.get).toHaveBeenCalledWith('users', {
+    params: {
+      'filter[location]': 1,
+      limit: 5,
+      page: 1,
+      sort: 'created',
+    },
+  });
+});
```
Comment (craigpaul) [inline src/resources/user.test.ts:230]: As I mentioned above:

> What is the reasoning behind not supplying both parameters if they are both supplied? I don't think this package should be laying down rules like this.

**Review by craigpaul — CHANGES_REQUESTED** [review]

File: src/resources/service.test.ts (hunk 2 above)
Comment (olya-morozova) [reply src/resources/service.test.ts]: I have removed the unnecessary logic and the associated tests

**Review by craigpaul — APPROVED** [review]

**Pull request merged**